### PR TITLE
update operate

### PR DESCRIPTION
**Pull request by katamaks**

### CODE REVIEW

File: developerportal/operate/index.md [hunk 1]
```diff
@@ -31,16 +28,21 @@ In this section you can view the graphs of the App about:
...
 At this section you can view the following information about the running requests:
 
-*   User
-*   Action
-*   Type
-*   Duration
+*   User - the user that is running the request
+*   Action - the currently running actions
+*   Type - the action type that is running
+*   Duration - how long the request is running until the **Refresh** button has been clicked
```
Comment (jtwaleson) [inline developerportal/operate/index.md]: replace "request" by "action"

remove "until the **Refresh** button has been clicked"

how long the request is running -> for how long the action has been running

File: developerportal/operate/index.md [hunk 2]
```diff
@@ -7,20 +7,17 @@ tags: ["Operate","App","Developer Portal"]
...
 
-On this page you can monitor the performance and health of your App. This page is divided into three subpages:
+On this page you can monitor the performance and health of your App. 
 
-*   Trends
-*   Running now
-*   Cache
+For more information, see [Metrics](/developerportal/operate/metrics).
+
+This page is divided into three subpages described below:
```
Comment (jtwaleson) [inline developerportal/operate/index.md]: subpages -> tabs

Comment (katamaks) [inline developerportal/operate/index.md]: done

File: developerportal/operate/index.md (hunk 1 above)
Comment (katamaks) [inline developerportal/operate/index.md]: done

**Pull request merged**